dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -193,7 +240,15 @@ def refute_estimate(self):
             refute_list.append(refute)
 
         else:
-            for category in simulation_results.shape[1]:
+            # True Causal Effect list
+            causal_effect_list = list( causal_effect_map.values() )
+            # Iterating through the refutation for each category
+            for category in range(simulation_results.shape[1]):
```

Comment:
category is not very informative name here. How about validation_set_id?

Reply:
which key is that?

Reply:
In the above snippet, key refers to a key in the ```causal_effect_map```. I store data in the form:
```train_category: true_causal_effect``` for validation data